### PR TITLE
Bootstrap Project: Newsweek clone pull request for code review

### PR DESCRIPTION
**Pull request by aulbytj**
This branch house the development of a clone of the Newsweek page.  The page was built mainly using the Bootstrap CSS library and a tiny bit of javascript to add an additional effect with the navbar at scroll.

[pull request](https://app.gitkraken.com/glo/board/5e20b2dce2e30a00106f6471/card/5e422635184ba700102f3276)

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @aulbytj 
### You have done a good job so far :clap: However, there are a few issues you need to work on before your project could be approved.
- Reconfigure the live demo properly, showing 404
live app
- On more stories section, there is too much space on the left and right which is not required. You can achieve the required output by adding width and height to the div tag and allowing scroll
space
### Make these changes and resubmit for another review. Good luck.
- For assistance, contact me on slack @iamraymond
- Happy coding!

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello Team,
### Following from previous reviews, you did not implement all the required changes. Kindly do so and resubmit for another code review.  Good luck.
- For assistance, contact me on slack @iamraymond
- Happy coding!

**Comment by davidmrtz-dev** [comment]
Pull Request under review by David Elí 👨‍💻

**Comment by davidmrtz-dev** [comment]
### **Status: Changes required.** 🛑 👀 🔍 
****
### Hi Team. 😄 
### Very good job so far. 🎊 
**But Before move on let's solve a couple of issues.**  💯 👨‍💻  

****

- [x] Blank space. 
Like the previous TSE said: On the large views "lg" and "xl" size there's a big blank space on the sides of "more stories".

Screen Shot 2020-03-05 at 9 56 41 PM


****
### Optional.

- [x] Navbar.

On "md" and "sm" your navbar look pretty nice, that sidebar using javascript looks awesome, congratulations for that, but on "lg" and "xl" sized there's a little square on the fixed navbar, it would look better without that square.

Screen Shot 2020-03-05 at 10 02 24 PM

****

### **Please kindly make the changes required and submit a new code review once you are ready!** 🙏 

Happy coding! 😄 💻  
If you have any questions or need help, you can message me on slack (@ David Eli Martinez Garcia).

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!-- @format -->
+
+# Project Name
+
+## Responsive Desgin and CSS Framework
```
Comment (euqueme) [inline README.md]: - [x] The project name is Using Bootstrap please change it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,3945 @@
...
+                          data-srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.webp?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6"
+                          srcset="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.webp?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6">
+                      <img
+                          data-src="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6"
+                          alt="BTS 'Map of the Soul: 7' Album Dominates Top 20 on iTunes U.S. Charts"
+                          title="BTS 'Map of the Soul: 7' Album Dominates Top 20 on iTunes U.S. Charts"
+                          class="w-100 h-auto small lazysize lazyloaded"
+                          src="https://d.newsweek.com/en/full/1568628/bts-map-soul-7-album-dominates-top-20-itunes-us-charts.jpg?w=397&amp;h=265&amp;f=fc1f58447f08f40c13983c98b9d155e6"
+                          width="397" height="265">
+                      </picture>
```
Comment (euqueme) [inline index.html]: - [x] All style selector and media queries should be declared in the CSS files

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @aulbytj :handshake: 

### Thank you for doing what was requested by previous reviewers, your project looks good but please address to the comments I left on your code :point_down: 

#### :no_good_woman: inline styling

- [x] I didn't put comments on all your code but please look wherever (especially `` tags)  you are using width, height, margin, padding or any style selectors and media queries directly on your HTML and replace them with either bootstrap classes or CSS classes.

### Please :pray:  do the changes :man_technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any question? ask me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

File: README.md (hunk 1 above)
Comment (aulbytj) [reply README.md]: done

File: index.html (hunk 2 above)
Comment (aulbytj) [reply index.html]: done

**Comment by uryelah** [comment]
Hello **Aulbourn Knowles**, I'm Sarah, your last reviewer for this PR 👩‍🚀 

You did an outstanding job fixing the required CR changes and styling the page 🎇 🥇 

### Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

-------
### Optional
- [ ] [OPTIONAL] If you want your project to look even closer to the original site, you can use `scrollbar-width` to make that scroll bar to the right of opinion to disappear.
[Check this CSS tricks articles on it. ](https://css-tricks.com/almanac/properties/s/scrollbar-width/)

-------

**Keep being an awesome dev and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by aulbytj** [comment]
Awesome!🕺🏾🙌🏾🎉

**Pull request merged**